model/DataQuery.php

```diff
@@ -13,6 +13,8 @@
  */
 class DataQuery {
 
+	public $lastAlias;
```

Comment:
Query parameters aren't really the right place for it either. See my comment below. :D